packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js

```diff
@@ -43,7 +43,7 @@ const CreateIssueButton = ({
       {...props}
       issueTitle={pageTitle && `Issue: ${pageTitle}`}
       issueBody={ISSUE_BODY}
-      labels={['bug']}
+      labels={['feedback', 'feedback-issue']}
```

Comment:
Just to verify, do we want this to apply to all sites? Or just the docs site?

Reply:
@jerelmiller ideally just docs site, as these  labels aren't used on developer / oss for tracking issues.  so if there is some IF / ELSE logic I can add here, I'm happy to do that.

Reply:
Ok! In that case, I think we should convert this into a prop for this component so that each site can specify its own set of labels. I think it probably still makes sense to default to `['bug']` if the prop isn't defined. Here are the changes I would make:

```js
const CreateIssueButton = ({ labels = ['bug'] /* <-- add `labels` prop }) => {
  // ...
  return (
    <GitHubIssueButton
      // ...
      labels={labels} // <--- use props instead of hard-coding the value here
    />
}

CreateIssueButton.propTypes = {
  labels: PropTypes.arrayOf(PropTypes.string) // <--- add prop type for new labels
}
```

We'll want to update the README as well to list the new `labels` prop for this component.

I'm going to dismiss my review until this is in place so we can get this working.